src/config/power-management.md

```diff
@@ -0,0 +1,29 @@
+# Power Management
+
+Void provides two approaches to power management:
+[acpid(8)](https://man.voidlinux.org/acpid.8) and
+[elogind(8)](https://man.voidlinux.org/elogind.8).
+
+## acpid
+
+The `acpid` service is installed and enabled by default. ACPI events are handled
+by `/etc/acpi/handler.sh`, which uses [zzz(8)](https://man.voidlinux.org/zzz.8)
+for suspend-to-RAM events.
+
+## elogind
+
+The `elogind` service is provided by the `elogind` package. By default,
+`elogind` listens for and processes ACPI events related to lid-switch activation
+and presses of the *power*, *suspend* and *hibernate* keys. This will conflict
```

Comment:
"and the ", perhaps?

Reply:
Mm, that sounds more awkward to me.

Reply:
"events related to [...] the *power* [...] keys" and "events related to [...] presses of the *power* [...] keys

Reply:
Okay, i'll change the sentence to:

> By default, `elogind` listens for, and processes, ACPI events related to lid-switch activation and the *power*, *suspend* and *hibernate* keys.

Reply:
PR updated.